game/src/debug/debug_plugin.rs

```diff
@@ -0,0 +1,15 @@
+use bevy::prelude::*;
+
+pub struct DebugPlugin;
```

Comment:
Please add documentation. You can do so by using the triple backslash notation:
```Rust
/// This describes what this plugin does
/// Maybe it also describe how to use it and expand on it
pub struct DebugPlugin;
```
[More info on rust documentation here](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html)

Reply:
Updated